mandatory/1-fix-functions.js

```diff
@@ -10,8 +10,8 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(mood){
```

Comment:
I think there should be a Boolean expression that evaluates to true or false. And I think mood can stay as function name, and isHappy as parameter. Then an if statement may work as follow-->> (isHappy === true) return 'I am happy'.

Reply:
Can you write the code you mean? I don’t get it.

Reply:
If you check your "if statement", how does 'isHappy' become a boolean expression without any boolean operators. Also you have both function name and parameter name the same ---mood